### PR TITLE
docs: update pd-configuration about upgrade

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-configuration-file.md [hunk 1]
```diff
@@ -257,12 +257,16 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 设置是否开启跨表 merge。
 + 默认值：true
 
-### `region-score-formula-version`
+### `region-score-formula-version` <span class="version-mark">从 v5.0 版本开始引入</span> 
 
 + 设置 Region 算分公式版本。
 + 默认值：v2
 + 可选值：v1，v2
```
Comment (qiancai) [inline pd-configuration-file.md:264]: @nolouch 这里能否可以增加下 v1，v2 的解释呢？可以帮助用户理解下面注意里提到的 “新策略”是什么含义

Comment (nolouch) [reply pd-configuration-file.md]: udpate

File: pd-configuration-file.md [hunk 2]
```diff
@@ -257,11 +257,15 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
+### `region-score-formula-version` <span class="version-mark">从 v5.0 版本开始引入</span> 
 
 + 设置 Region 算分公式版本。
 + 默认值：v2
-+ 可选值：v1，v2
++ 可选值：v1，v2。v2 相比于 v1，变化会更平滑，空间回收引起的调度抖动情况会得到改善。
+
+> **注意：**
+>
+> 4.x 升级上来默认不会自动开启新策略以保证升级后行为一致，若想切换使用需要手动通过 `pd-ctl` 设置切换。
```
Comment (TomShawn) [inline pd-configuration-file.md:268]:
```suggestion
> 如果是从 v4.0 升级至当前版本，默认不自动开启该算分公式新版本，以保证升级前后 PD 行为一致。若想切换算分公式的版本，使用需要手动通过 `pd-ctl` 设置切换，详见 [PD Control](/pd-control.md#config-show--set-option-value--placement-rules) 文档。
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: pd-configuration-file.md [hunk 3]
```diff
@@ -306,6 +310,10 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 默认值：3
 + PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息的末尾进行四舍五入的位数。例如流量 `100512` 会归约到 `101000`。默认值为 `3`。该配置替换了 `trace-region-flow`。
 
+> **注意：**
+>
+> 4.x 升级上来的保持和 `trace-region-flow` 设置一致，如果 `trace-region-flow` 为 false，则升级后 `flow-round-by-digit` 为 127。 否则为 3。
```
Comment (qiancai) [inline pd-configuration-file.md:315]:
```suggestion
> 如果是从 v4.x 升级至当前版本，升级后的 `flow-round-by-digit` 行为和升级前的 `trace-region-flow` 行为默认保持一致：如果升级前 `trace-region-flow` 为 false，则升级后 `flow-round-by-digit` 为 127；否则为 3。
```

File: pd-configuration-file.md [hunk 4]
```diff
@@ -306,6 +310,10 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 默认值：3
 + PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息的末尾进行四舍五入的位数。例如流量 `100512` 会归约到 `101000`。默认值为 `3`。该配置替换了 `trace-region-flow`。
 
+> **注意：**
+>
+> 如果是从 v4.x 升级至当前版本，升级后的 `flow-round-by-digit` 行为和升级前的 `trace-region-flow` 行为默认保持一致：如果升级前 `trace-region-flow` 为 false，则升级后 `flow-round-by-digit` 为 127；如果升级前 `trace-region-flow` 为 true，则升级后 `flow-round-by-digit` 为 3。
```
Comment (TomShawn) [inline pd-configuration-file.md:315]:
```suggestion
> 如果是从 v4.0 升级至当前版本，升级后的 `flow-round-by-digit` 行为和升级前的 `trace-region-flow` 行为默认保持一致：如果升级前 `trace-region-flow` 为 false，则升级后 `flow-round-by-digit` 为 127；如果升级前 `trace-region-flow` 为 true，则升级后 `flow-round-by-digit` 为 3。
```

**Pull request merged**